test/runtests.jl

```diff
@@ -1,136 +1,136 @@
 using SyntheticGrids
-using Base.Test
+using Test
 using JSON
 
 @testset "SyntheticGrids" begin
-    const SEED = 666
-    const GENPATH = joinpath(dirname(@__FILE__), "..", "data", "GenData.json")
-    const CENSUSPATH = joinpath(dirname(@__FILE__), "..", "data", "Census_data.dat")
-    const GENCOORDPATH = joinpath(dirname(@__FILE__), "..", "data", "Generator_coord.dat")
-    const GENDATAPATH = joinpath(dirname(@__FILE__), "..", "data", "Generator_data.dat")
-    const DUMMYPATH = joinpath(dirname(@__FILE__), "..", "data", "dummy.json")
-    const DUMMYPPC = joinpath(dirname(@__FILE__), "..", "data", "dummy.p")
-    const GRIDPATH = joinpath(dirname(@__FILE__), "..", "data", "testgrid.json")
+    SEED = 666
+    LATLIM = (38, 40)
+    LONGLIM = (-89, -88)
+    GENPATH = joinpath(dirname(@__FILE__), "..", "data", "GenData.json")
+    CENSUSPATH = joinpath(dirname(@__FILE__), "..", "data", "Census_data.dat")
+    GENCOORDPATH = joinpath(dirname(@__FILE__), "..", "data", "Generator_coord.dat")
+    GENDATAPATH = joinpath(dirname(@__FILE__), "..", "data", "Generator_data.dat")
+    DUMMYPATH = joinpath(dirname(@__FILE__), "..", "data", "dummy.json")
+    DUMMYPPC = joinpath(dirname(@__FILE__), "..", "data", "dummy.p")
+    GRIDPATH = joinpath(dirname(@__FILE__), "..", "data", "testgrid.json")
 
-    grid = Grid(SEED)
-    grid1 = Grid(SEED)
-
-    @testset "Basic grid creation" begin
-        place_loads_from_zips!(grid; latlim = (38, 40), longlim = (-89, -88))
-        @test length(buses(grid)) == 130
-        place_gens_from_data!(grid; latlim = (38, 40), longlim = (-89, -88))
-        @test length(buses(grid)) == 147
+    function baseline_grid(seed=666, latlim=(38, 40), longlim=(-89, -88))
+        grid = Grid(seed)
+        place_loads_from_zips!(grid; latlim=latlim, longlim=longlim)
+        place_gens_from_data!(grid; latlim=latlim, longlim=longlim)
         connect!(grid)
-        @test sum(adjacency(grid)) == 358
-        count = SyntheticGrids.count_bus_type(grid)
-        @test count == (130, 17)
         create_lines!(grid)
-        @test grid.trans_lines[1].capacity == 4900
-        @test length(trans_lines(grid)) == 179
-        @test test_connectivity(grid.bus_conn, false)
-        @test total_links(grid.bus_conn) == 179
-        @test mean_node_deg(grid.bus_conn) ≈ 2.435374149659
-        @test mean_shortest_path(adjacency(grid)) ≈ 6.764218612615115
-        @test mean_shortest_path(adjacency(grid), distance(buses(grid))) ≈ 133.1803034218734
-        @test robustness_line(grid.bus_conn, 10) > 1
-        @test robustness_node(grid.bus_conn, 10) > 1
-        place_loads_from_zips!(
-            grid1;
-            latlim = (38, 40),
-            longlim = (-89, -88),
-        )
-        place_gens_from_data!(
-            grid1;
-            latlim = (38, 40),
-            longlim = (-89, -88),
-        )
-        connect!(grid1)
-        create_lines!(grid1)
-        @test grid == grid1
```

Comment:
This test should not be removed. It tests both that the seeding is working fine and that the custom comparison between grid objects is working as well.